### PR TITLE
Updates/dec 27 vivek

### PR DESCRIPTION
**Pull request by vivek1214-git**
view and update form

### CODE REVIEW

File: app/Http/Controllers/CounselorSessionController.php [hunk 1]
```diff
@@ -22,35 +23,45 @@ public function store(Request $request)
...
+        return redirect('/students');
     }
 
-    public function show(Request $request)
+    public function show($id)
     {
-        $userInformation = CounselorSession::where('user_id', $request->id)->get()->first();
-        $user = User::find($request->id);
-        return view('counselor-session', compact('userInformation', 'user'));
+        $userInformation = CounselorSession::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/CounselorSessionController.php:32]: change variable name to meaningfull $userInformation-> $counselorSession

File: app/Http/Controllers/CounselorSessionController.php [hunk 2]
```diff
@@ -22,35 +23,45 @@ public function store(Request $request)
...
+            return view('counselor-session-form', compact('user', 'actionUrl'));
+        }
     }
 
-    public function update(Request $request)
+    public function edit($id)
     {
-        $userInformation = CounselorSession::find($request->user_information_id);
+        $actionUrl = "/students/$id/counselor-session-form";
+        $userInformation = CounselorSession::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/CounselorSessionController.php:47]: change variable name same as above

File: app/Http/Controllers/CounselorSessionController.php [hunk 3]
```diff
@@ -22,35 +23,45 @@ public function store(Request $request)
...
+        $user = User::find($id);
+        return view('counselor-session-form', compact('user', 'userInformation', 'actionUrl'));
+    }
+
+    public function update(Request $request, $id)
+    {
+        $userInformation = CounselorSession::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/CounselorSessionController.php:54]: change variable same as above

File: app/Http/Controllers/ExtraScoreController.php [hunk 4]
```diff
@@ -20,33 +21,41 @@ public function store(Request $request)
...
+        return redirect('/students');
     }
 
-    public function show(Request $request)
+    public function show($id)
     {
-        $userInformation = ExtraScore::where('user_id', $request->id)->get()->first();
-        $user = User::find($request->id);
-        return view('extra-score', compact('userInformation', 'user'));
+        $userInformation = ExtraScore::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/ExtraScoreController.php:30]: change variable same as above to meaningfull

File: app/Http/Controllers/ExtraScoreController.php [hunk 5]
```diff
@@ -20,33 +21,41 @@ public function store(Request $request)
...
+            return view('extra-score-form', compact('user', 'actionUrl'));
+        }
     }
 
-    public function update(Request $request)
+    public function edit($id)
     {
-        $userInformation = ExtraScore::find($request->user_information_id);
+        $actionUrl = "/students/$id/extra-score-form";
+        $userInformation = ExtraScore::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/ExtraScoreController.php:43]: change variable name to meaninfull

File: app/Http/Controllers/ExtraScoreController.php [hunk 6]
```diff
@@ -20,33 +21,41 @@ public function store(Request $request)
...
+        $user = User::find($id);
+        return view('extra-score-form', compact('user', 'userInformation', 'actionUrl'));
+    }
+
+    public function update(Request $request, $id)
+    {
+        $userInformation = ExtraScore::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/ExtraScoreController.php:50]: change variabe name to meaningfull

File: app/Http/Controllers/SelfieController.php [hunk 7]
```diff
@@ -17,33 +18,41 @@ public function store(Request $request)
...
+        Session::flash('message', 'Data inserted successfully !');
+        return redirect('/students');
     }
-    public function show(Request $request)
+    public function show($id)
     {
-        $userInformation = Selfie::where('user_id', $request->id)->get()->first();
-        $user = User::find($request->id);
-        return view('selfie', compact('userInformation', 'user'));
+        $userInformation = Selfie::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/SelfieController.php:27]: change variable name same as above

File: app/Http/Controllers/SelfieController.php [hunk 8]
```diff
@@ -17,33 +18,41 @@ public function store(Request $request)
...
+            return view('selfie-form', compact('user', 'actionUrl'));
+        }
     }
 
-    public function update(Request $request)
+    public function edit($id)
     {
-        $userInformation = Selfie::find($request->user_information_id);
+        $actionUrl = "/students/$id/selfie-form";
+        $userInformation = Selfie::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/SelfieController.php:42]: change variable name same as above

File: app/Http/Controllers/SelfieController.php [hunk 9]
```diff
@@ -17,33 +18,41 @@ public function store(Request $request)
...
+        $user = User::find($id);
+        return view('selfie-form', compact('user', 'userInformation', 'actionUrl'));
+    }
+
+    public function update(Request $request, $id)
+    {
+        $userInformation = Selfie::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/SelfieController.php:49]: change variable name same as above

**Review by sumedh-mohod — COMMENTED** [review]
format all files and remove extra spacing

**Comment by ithqindia** [comment]
https://github.com/ithqindia/metroclub/pull/26

**Pull request closed without merging**